tasks/test_syntax.js

```diff
@@ -112,9 +112,6 @@ function assertSrcContents() {
     // Forbidden in IE in any context
     var IE_BLACK_LIST = ['classList'];
 
-    // not implemented in FF, or inconsistent with others
-    var FF_BLACK_LIST = ['offsetX', 'offsetY'];
```

Comment:
Has this new behavior been tested in FF? Looks like this just used in mapbox. According to https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/offsetX perhaps this "not implemented in FF" is long long obsolete, but I'm not sure what the "inconsistent with others" comment was about.

Reply:
I tested it in FF. And it works.